### ISSUE TITLE
Call stats not loading

### ISSUE THREAD

**Issue by terrymason** [bdm1981/finesseGadgets#27]
Running UCCX 11, and I'm having a bear of a time getting call history to show.  I've uploaded all files to both my uccx pub and sub.  I've removed the comments from these lines:
    <script type="text/javascript" src="/desktop/assets/js/jquery.min.js"></script>
    <script type="text/javascript" src="/desktop/assets/js/finesse.js"></script>
and I've restarted finesse services (eventually i restarted the entire cluster).

Testing in IE 11 and firefox 47

At first no stats show, but then when I receive a call, (referencing the pic below) the stats in section 1 all popup as 0.  They stay 0 no matter how many calls I answer.  sections 2,3,and 4 never populate with anything

https://camo.githubusercontent.com/8269b4476c5dd1ad55ce1071b80cdf76327049d1/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f62646d2d66696c65732f73637265656e73686f74732f63616c6c486973746f72794f766572766965772e6a7067

Any help would be appreciated.

**Comment by bdm1981** [comment]
@terrymason are you loading any other custom gadgets?

**Comment by terrymason** [comment]
No, this is my very first custom gadget. 
It's actually a brand new UCCX deployment.  It's not production yet.

**Comment by bdm1981** [comment]
@terrymason Are these inbound calls to the actual UCCX queue or to the agent extension? What happens when an outbound call is initiated from the agents line? 

Can you send me a copy of your callHistoryGadget.xml and callHistoryGadget.js files?

**Comment by terrymason** [comment]
So, it appears that calls directly to and from the extension work fine (they show up in call history).  Calls coming in through the call center don't show up, and the "call back" button doesn't seem to work at all.

Thank you very much for your help on this, and for providing this great tool.

[gadgets.zip](https://github.com/bdm1981/finesseGadgets/files/739884/gadgets.zip)

**Comment by bdm1981** [comment]
Your gadget files look good. Use firefox and open the developer web console while receiving a call from the call center and direct to the agent extension. If you could capture this and send it over, that might give me some insight into whats going on. Logging in IE is terrible so I would use firefox to look at this data :). Finesse logs a TON of data. Most of it will not pertain to this gadget but the context may give me a clue.

Thanks,

- Brad

**Comment by terrymason** [comment]
Brad,
What's the best way to get these logs to you?  I don't think I can share that info on a public forum.

Thanks again.

**Comment by bdm1981** [comment]
@terrymason. no worries, email them to bdm@bdmenterprises.com

**Comment by bdm1981** [comment]
Hi Terry,

I just downloaded a fresh copy of the gadget from github, uncommented the
lines for the onbox jquery and finesse libraries and uploaded to my UCCX
11.5 system as a 3rd party gadget it and ran like a champ. I enabled
automatic wrap-up time and that didn't have any effect on it either.

I just scheduled a UCCX 11.0 session on dCloud and will give it a shot
there as well. hopefully I can get it to act up!

I will let you know what the outcome is.

- Brad

--
Brad McAllister
CCIE# 40627
BDM Enterprises, Inc.
(847)243-6921
